### PR TITLE
PT-BR - Translation of TF 2.0 keras tutorials

### PR DESCRIPTION
**Pull request by brunapinos**

### CODE REVIEW

File: site/pt/beta/tutorials/keras/basic_regression.ipynb [hunk 1]
```diff
@@ -0,0 +1,865 @@
+{
```
Comment (victorcmoura) [inline site/pt/beta/tutorials/keras/basic_regression.ipynb]: <p>Typo:</p><blockquote>essas estatísticas serão usadas também <strong>para</strong> normalizar o conjunto de teste.</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/839/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Comment (victorcmoura) [inline site/pt/beta/tutorials/keras/basic_regression.ipynb]: <p>Shouldn't this be translated too?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/839/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Comment (victorcmoura) [inline site/pt/beta/tutorials/keras/basic_regression.ipynb]: <p>Typo:</p><blockquote>Vamos atualizar o&nbsp;<code>model.fit</code>&nbsp;para que pare <strong>automaticamente</strong> o treinamento</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/839/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>

Comment (victorcmoura) [inline site/pt/beta/tutorials/keras/basic_regression.ipynb]: <p>Typos:</p><blockquote>Mean Sqaured Error(MSE), é uma função comum de&nbsp;<em>loss</em>&nbsp;usada para problemas de regressão (diferentes <strong>funções</strong> de&nbsp;<em>loss</em>&nbsp;são usadas para problemas de classificação)</blockquote><blockquote>[...]</blockquote><blockquote>Similarmente, as métricas de evolução usadas na <strong>regressão</strong> são diferentes da classificação</blockquote><blockquote>[...]</blockquote><blockquote>Se não <strong>possuir</strong> muitos dados de treinamento, uma técnica é preferir uma pequena rede com poucas camadas para evitar&nbsp;<em>overfitting</em></blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/839/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='56'/>

File: site/pt/beta/tutorials/keras/basic_text_classification.ipynb [hunk 2]
```diff
@@ -0,0 +1,726 @@
+{
```
Comment (victorcmoura) [inline site/pt/beta/tutorials/keras/basic_text_classification.ipynb]: <p>I believe there are a few grammar mistakes. The pronoun "Ele", which refers to "base de dados", should be in the feminine form, since it referrs to a feminine noun. Despite that, "avaliações" is a plural noun and there are a few verbal disagreements when referring to it:</p><blockquote>A base de dados vem empacotada com TensorFlow. <strong>Ela</strong> já vem <strong>pré-processada</strong> de forma que as avaliações (sequências de palavras) <strong>foram</strong> <strong>convertidas</strong> em sequências de inteiros, onde cada inteiro representa uma palavra específica no dicionário.</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/839/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (victorcmoura) [inline site/pt/beta/tutorials/keras/basic_text_classification.ipynb]: <p>I would translate with a few differences, to match the original text:</p><blockquote>As avaliações—<strong>os</strong>&nbsp;<em>arrays</em>&nbsp;de inteiros— <strong>devem</strong> ser <strong>convertidas</strong> em tensores (<em>tensors</em>) antes de alimentar a rede neural.</blockquote><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/839/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Comment (victorcmoura) [inline site/pt/beta/tutorials/keras/basic_text_classification.ipynb]: <p>Should this repetition occur?</p><blockquote>A primeira camada é uma camada&nbsp;<code>Embedding</code>&nbsp;layer (<code><em>Embedding</em></code><em>&nbsp;layer</em>).</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/839/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Comment (victorcmoura) [inline site/pt/beta/tutorials/keras/basic_text_classification.ipynb]: <p>Typos:</p><blockquote>Entretanto, isso faz com que a rede seja <strong>computacionalmente</strong> mais custosa e pode levar <strong>ao</strong> aprendizado de padrões não desejados</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/839/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Comment (victorcmoura) [inline site/pt/beta/tutorials/keras/basic_text_classification.ipynb]: <p>I would translate differently:</p><blockquote>Já que é um problema de classificação binário e o modelo tem, <strong>como</strong> saída, uma probabilidade</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/839/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Comment (victorcmoura) [inline site/pt/beta/tutorials/keras/basic_text_classification.ipynb]: <p>Instead of "aproximação", I would translate "approach" into "abordagem". There are also a few typos:</p><blockquote><strong>Esta</strong> é uma <strong>abordagem</strong> ingênua que conseguiu uma acurácia de 87%. <strong>Com abordagens mais avançadas</strong>, o modelo deve chegar em 95%.</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/839/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

File: site/pt/beta/tutorials/keras/basic_regression.ipynb [hunk 3]
```diff
@@ -488,9 +488,9 @@
         "id": "BuiClDk45eS4"
       },
       "source": [
-        "This normalized data is what we will use to train the model.\n",
+        "O dado normalizado é o que nós usaremos para treinar o modelo.\n",
         "\n",
-        "Caution: The statistics used to normalize the inputs here (mean and standard deviation) need to be applied to any other data that is fed to the model, along with the one-hot encoding that we did earlier.  That includes the test set as well as live data when the model is used in production."
+        "Atenção: A estatística usada para normalizar as entradas aqui (média e desvio padrão) precisa ser aplicada em qualquer outro dado que alimenta o modelo, junto com o codificação one-hot que fizemos mais cedo.  Isso inclui o conjunto de teste, assim como dados que o modelo usará em produção."
```
Comment (lamberta) [inline site/pt/beta/tutorials/keras/basic_regression.ipynb]: 'Caution:', 'Note:', and some other tags are used for styling on the website. They should not be translated.
I can fix this one
